pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -22,51 +28,58 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/scheme"
 	"k8s.io/kubernetes/pkg/kubectl/util/templates"
 
-	v1 "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
+	imageclient "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
 	"github.com/openshift/library-go/pkg/image/imageutil"
 	"github.com/openshift/library-go/pkg/operator/resource/retry"
+
 	"github.com/openshift/oc/pkg/cli/rsync"
 )
 
 var (
 	mustGatherLong = templates.LongDesc(`
 		Launch a pod to gather debugging information
 
-		This command will launch a pod in a temporary namespace on your
-		cluster that gathers debugging information, using a copy of the active
-		client config context, and then downloads the gathered information.
+		This command will launch a pod in a temporary namespace on your cluster that gathers 
+		debugging information and then downloads the gathered information.
 
 		Experimental: This command is under active development and may change without notice.
 	`)
 
 	mustGatherExample = templates.Examples(`
-		# gather default information using the default image and command, writing into ./must-gather.local.<rand>
+		# gather information using the default plug-in image and command, writing into ./must-gather.local.<rand>
 		  oc adm must-gather
 
-		# gather default information with a specific local folder to copy to
+		# gather information with a specific local folder to copy to
 		  oc adm must-gather --dest-dir=/local/directory
 
-		# gather default information using a specific image, command, and pod-dir
+		# gather information using multiple plug-in images 
+		  oc adm must-gather --image=quay.io/kubevirt/must-gather --image=quay.io/openshift/origin-must-gather
+
+		# gather information using a specific image stream plug-in 
+		  oc adm must-gather --image-stream=openshift/must-gather:latest
+
+		# gather information using a specific image, command, and pod-dir
 		  oc adm must-gather --image=my/image:tag --source-dir=/pod/directory -- myspecial-command.sh
 	`)
 )
 
 func NewMustGatherCommand(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
 	o := NewMustGatherOptions(streams)
-	rsyncCommand := rsync.NewCmdRsync(rsync.RsyncRecommendedName, "", f, streams)
 	cmd := &cobra.Command{
 		Use:     "must-gather",
 		Short:   "Launch a new instance of a pod for gathering debug information",
 		Long:    mustGatherLong,
 		Example: mustGatherExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			kcmdutil.CheckErr(o.Complete(f, cmd, args))
-			kcmdutil.CheckErr(o.Run(rsyncCommand))
+			kcmdutil.CheckErr(o.Initialize(f))
+			kcmdutil.CheckErr(o.Complete(cmd, args))
+			kcmdutil.CheckErr(o.Run())
```

Comment:
The CLI trio is: `Complete`, `Validate`, `Run` _always_.

Reply:
Done